website/versioned_docs/version-v1.23/quickstart-guide/deb.md

```diff
@@ -34,4 +34,4 @@ For more information about its configuration, please take a look at [systemd con
 
 Find out more about:
 
-- [getting logs](../configuration/observability.md#binary-executable-logs).
```

Comment:
Should the linter run on versioned docs too? Just saying because having versioned docs seems less useful if we modify them.

Reply:
What linter?

Reply:
Ah it's a configuration `onBrokenAnchors: 'throw'`, not a linter.

Reply:
And that configuration applies to all docs